README.md

```diff
@@ -1,3 +1,8 @@
 # Google Student Training in Engineering Program
 
-This repo contains the STEP capstone project: "Voter Central".
\ No newline at end of file
+This repo contains the STEP capstone project: "Voter Central".
+
+<ul>
+  <li>Live server: go/voter-central</li>
+  <li>Design doc: go/voter-central-dd</li>
```

Comment:
nit: make these clickable
http://google.github.io/styleguide/docguide/style.html#links
```suggestion
  <li>Live server: [go/voter-central](http://go/voter-central)</li>
  <li>Design doc: [go/voter-central-dd](http://go/voter-central-dd)</li>
```

Reply:
Will do, thanks!

Reply:
Oh actually, I think you have to format the list as a markdown list (either numbered or bullet) for this to render properly: http://google.github.io/styleguide/docguide/style.html#lists

Reply:
Yeah, I realized the same thing. I've updated the latest commit

Reply:
Thanks!